packages/authenticator/amplify_authenticator/lib/src/l10n/authenticator_localizations.dart

```diff
@@ -64,4 +67,13 @@ abstract class AuthenticatorLocalizations {
   static AuthenticatorCountryLocalizations countriesOf(BuildContext context) {
     return AuthenticatorCountryLocalizations.of(context) ?? _countriesFallback;
   }
+
+  /// Retrieves the [AuthenticatorInstructionLocalizations] instance, falling back
+  /// to English if unavailable for this locale.
+  static AuthenticatorInstructionLocalizations instructionOf(
```

Comment:
Should these be `TotpInstructions` instead of `Instruction`?

Reply:
Or maybe just `Totp` and group everything TOTP related. There's also `totpCodeInstruction` in the confirm sign in types. Seems like we need to consolidate.

Reply:
But then we have `totpCodeInstruction` which is not in this class.. that seems a bit weird, no?